.dependabot/config.yml

```
@@ -0,0 +1,37 @@
+version: 1
+update_configs:
+  # Keep docker modules up to date in root, batching pull requests weekly
```

From our Slack conversation, I think this is a more accurate comment:
```suggestion
  # Keep base image up to date in root, batching pull requests weekly
```